### DISCUSSION TITLE
Cursor style as block does not work, on macOS

### DISCUSSION THREAD

**Discussion by applebait** [ghostty-org/ghostty#3601] [Q&A] [answered]
I think I've read all the titles on the Issues section and didn't find it.
Cursor thickness works, of course, for bar style but if I change to block it goes to bar, and it loses the thickness value.
Underline also does not work. Basically only bar works.

TIA

**Comment by lucsky** [comment, marked as answer]
Per the documentation: 

> Note that shell integration will automatically set the cursor to a bar at a prompt, regardless of this configuration. You can disable that behavior by specifying shell-integration-features = no-cursor or disabling shell integration entirely.

See https://ghostty.org/docs/config/reference#cursor-style

**Comment by applebait** [comment]
Done then! Thanks!
